cmd/update/project.go

```diff
@@ -84,10 +82,16 @@ Usage
 )
 
 func updateProjectsFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
-	projectSpec, err := project.DefaultProjectConfig.GetProjectSpec(config.GetConfig().Project)
+	projectSpec, err := project.DefaultProjectConfig.GetProjectSpec()
+
 	if err != nil {
 		return err
 	}
+
+	if project.DefaultProjectConfig.ID != "" {
```

Comment:
I think we no longer need to check if this is empty since `GetProjectSpec` should now check and error